### ISSUE TITLE
How do you encode an array?

### ISSUE THREAD

**Issue by dustinfarris** [livebook-dev/vega_lite#14]
e.g.

```
"encoding": {
    "x": {"timeUnit": "yearmonthdate", "field": "date"},
    "tooltip": [
      {"timeUnit": "yearmonthdate", "field": "date"},
      {"field": "temp_max", "type": "quantitative"},
      {"field": "temp_min", "type": "quantitative"}
    ]
  },
```

**Comment by jonatanklosko** [comment]
Hey @dustinfarris, good catch! I've just created #15 to address it :)

**Comment by jonatanklosko** [comment]
Released in v0.1.1!

**Comment by dustinfarris** [comment]
Thanks! ❤️
